importing/views.py

```diff
@@ -45,3 +64,157 @@ class DataImportDeleteView(CustomDeleteView):
+
+        **Permissions**: Users can only upload data for stations they have
+        `change_station` permission for.
+
+        **File Upload**: Send as multipart/form-data with all parameters in the body.
+
+        **Reprocess**: Set to `true` to reprocess the data after import.
+        """,
+        request={
+            "multipart/form-data": {
+                "type": "object",
+                "properties": {
+                    "station": {
+                        "type": "string",
+                        "description": "Station code",
+                    },
+                    "format": {
+                        "type": "integer",
+                        "description": "Format ID",
+                    },
+                    "rawfile": {
+                        "type": "string",
+                        "format": "binary",
+                        "description": "Data file to upload",
+                    },
+                    "visibility": {
+                        "type": "string",
+                        "enum": ["public", "private"],
+                        "default": "private",
+                        "description": "Visibility level",
+                    },
+                    "reprocess": {
+                        "type": "boolean",
+                        "default": False,
+                        "description": "Reprocess data after import",
+                    },
+                    "observations": {
+                        "type": "string",
+                        "description": "Additional observations",
+                    },
+                },
+                "required": ["station", "format", "rawfile"],
+            }
+        },
+        responses={
+            201: OpenApiResponse(
+                response=DataImportUploadResponseSerializer,
+                description="Data import created successfully",
+                examples=[
+                    OpenApiExample(
+                        "Successful upload",
+                        value={
+                            "data_import_id": 123,
+                            "station": "CAR_02_HC_01",
+                            "format": 47,
+                            "rawfile": "/media/imports/data_2024.csv",
+                            "visibility": "private",
+                            "reprocess": True,
+                            "date": "2024-12-09T10:30:00Z",
+                            "status": "N",
+                            "status_display": "Not queued",
+                        },
+                    )
+                ],
+            ),
+            400: OpenApiResponse(description="Invalid request parameters or file"),
+            403: OpenApiResponse(
+                description="User does not have permission to upload data."
+            ),
+            404: OpenApiResponse(description="Station or format not found"),
+        },
+        tags=["data-import"],
```

Comment:
I think you called this `importing` somewhere else, so we should use the same thing to make it appear in the right place in the API documentation. 
```suggestion
        tags=["importing"],
```

This makes me wonder why in measurements we have `download` and `download{format}`. There should be only one, right?

<img width="1627" height="956" alt="Image" src="https://github.com/user-attachments/assets/5d94a5a4-33dc-4711-bda1-91b4ec90907f" />